### PR TITLE
Add multiple resources to Snaps resources page

### PR DESCRIPTION
**Pull request by Montoya**
# Description

Adds multiple resources to the Snaps resources page and updates the formatting of descriptions for each one. Also updates the description for the Snaps Simulator resource.

## Issue(s) fixed

N/A

## Checklist

Complete this checklist before merging your PR:

- [ ] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page. N/A
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

File: snaps/learn/resources.md [hunk 1]
```diff
@@ -10,30 +10,52 @@ View the following Snaps resources in addition to this documentation site.
...
+- [Text Transformer](https://github.com/Montoya/transformer-snap) - Transforms 
+  strings into unicode bold, italic, and strikethrough characters for use on social 
+  media. Demonstrates [interactive JSX UI](../features/custom-ui/index.md) in a home page.
+- [Social Names](https://github.com/Montoya/social-names-snap) - Adds Farcaster and 
+  Lens handles to the send flow and petnames using 
+  [custom name resolution](../features/custom-name-resolution.md).
 
 ## Developer tools
 
 - [Template Snap](https://github.com/MetaMask/template-snap-monorepo) - A rich template that
```
Comment (FrederikBolding) [inline snaps/learn/resources.md:47]: Can we add `snaps-jest`?

File: snaps/learn/resources.md [hunk 2]
```diff
@@ -10,30 +10,52 @@ View the following Snaps resources in addition to this documentation site.
 ## Primary resources
 
 - [Snaps homepage](https://metamask.io/snaps/)
-- [Snaps directory](https://snaps.metamask.io/) - A directory of allowlisted Snaps you can try in the MetaMask extension.
-- [SIPs](https://github.com/MetaMask/SIPs) - Suggest new Snaps APIs with Snaps Improvement Proposals.
-- [Builder Engagement Program](https://go.metamask.io/snaps-builders) - Connect with the Snaps builder engagement team and get support while building your Snap.
+- [Snaps directory](https://snaps.metamask.io/) - A directory of allowlisted
```
[the remark this answers is not in the document]
Comment (Montoya) [reply snaps/learn/resources.md:13]: I noticed some lines had hard line breaks and decided to make the whole file consistent. Will wait for @alexandratran to let me know if it is preferable to have them or remove them. I am fine to revert all the line breaks instead.

File: snaps/learn/resources.md (hunk 1 above)
Comment (Montoya) [reply snaps/learn/resources.md:47]: Will do.

**Review by FrederikBolding — APPROVED** [review]

File: snaps/learn/resources.md [hunk 3]
```diff
@@ -10,30 +10,55 @@ View the following Snaps resources in addition to this documentation site.
...
 - [Template Snap](https://github.com/MetaMask/template-snap-monorepo) - A rich template that
   includes TypeScript/React and vanilla JavaScript options, a CLI for building, packaging, and
   deploying your Snap, and a companion dapp UI you can build on.
-- [Snaps Simulator](https://metamask.github.io/snaps/snaps-simulator/latest) - A tool for simulating
-  Snaps in the browser, streamlining the development process.
 - [Test Snaps](https://github.com/MetaMask/snaps/tree/main/packages/test-snaps) - A collection of
   test Snaps and [a dapp for evaluating them](https://metamask.github.io/snaps/test-snaps/latest/).
+- [snaps-jest](https://www.npmjs.com/package/@metamask/snaps-jest) - A Jest preset for end-to-end 
+  testing MetaMask Snaps, including a Jest environment, and a set of Jest matchers. 
+  [How to use snaps-jest](../how-to/test-a-snap.md).
```
Comment (alexandratran) [inline snaps/learn/resources.md:54]:
```suggestion
- [`snaps-jest`](https://www.npmjs.com/package/@metamask/snaps-jest) - A Jest preset for end-to-end 
  testing MetaMask Snaps, including a Jest environment, and a set of Jest matchers.
  See [how to test a Snap](../how-to/test-a-snap.md) using `snaps-jest`.
```

**Review by alexandratran — APPROVED** [review]

File: snaps/learn/resources.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexandratran) [reply snaps/learn/resources.md:13]: We typically write with a [100 column limit](https://docs-template.consensys.io/contribute/format-markdown#column-limit) for readability, but haven't been enforcing this for other contributors, resulting in some inconsistencies. I think the new linter will raise warnings for lines longer than 200 columns.

**Pull request merged**